pkg/cluster/info.go

```diff
@@ -220,3 +221,25 @@ func MergeImageOverrides(imageOverrides map[string]string, localImageOverrides [
 
 	return imageOverrides, nil
 }
+
+func (c *Info) GetBroker(namespace string) (*v1alpha1.Broker, error) {
+	broker := &v1alpha1.Broker{}
+	err := c.ClientProducer.ForGeneral().Get(
+		context.TODO(), controllerClient.ObjectKey{
+			Namespace: namespace,
+			Name:      brokercr.Name,
+		}, broker)
+
+	return broker, errors.Wrap(err, "error retrieving Broker")
+}
+
+func (c *Info) GetClusters(namespace string) ([]submarinerv1.Cluster, error) {
+	clusters := &submarinerv1.ClusterList{}
+
+	err := c.ClientProducer.ForGeneral().List(context.TODO(), clusters, controllerClient.InNamespace(namespace))
+	if err != nil {
+		return nil, errors.Wrap(err, "error retrieving Clusters")
+	}
+
+	return clusters.Items, nil
+}
```

Comment:
These should probably be declared after `OperatorNamespace` as they're also instance functions.